utils/data-file.c

```diff
@@ -132,12 +133,14 @@ int read_task_txt_file(char *dirname, bool needs_session, bool sym_rel_addr)
 			create_task(&task, true, needs_session);
 		}
 		else if (!strncmp(line, "SESS", 4)) {
-			if (!needs_session)
-				continue;
-
```

Comment:
I think you should keep this check as is.  If `needs_session` is false, no need to set kaddr IMHO.